monai/data/nifti_reader.py

```diff
@@ -28,7 +28,7 @@ def __init__(
         self,
         image_files: Sequence[str],
         seg_files: Optional[Sequence[str]] = None,
-        labels: Optional[Sequence[Union[int, float]]] = None,
+        labels: Optional[Sequence[float]] = None,
```

Comment:
May I know why you change from `Union[int, float]` to `float`?

Reply:
https://www.python.org/dev/peps/pep-0484/#the-numeric-tower

> Rather than requiring that users write import numbers and then use numbers.Float etc., this PEP proposes a straightforward shortcut that is almost as effective: when an argument is annotated as having type float, an argument of type int is acceptable